tests/integration/cattletest/core/common_fixtures.py

```diff
@@ -477,6 +477,15 @@ def auth_check(schema, id, access, props=None):
     return 1
 
 
+def resource_action_check(schema, id, actions):
+    keys = schema.types[id].resourceActions.keys()
```

Comment:
This test isn't comprehensive enough.  It won't test if there is a process defined in the schema, but you haven't listed it in the actions parameter.  This behavior follows how the existing auth checks work.  In fact it would be better to just extend the auth_check function.  That way all authorization checks can be in the same place.